CMake/config.cmake

```diff
@@ -43,6 +43,7 @@ mark_as_advanced(TTK_CELL_ARRAY_LAYOUT)
 option(TTK_ENABLE_MPI "Enable MPI support" FALSE)
 if (TTK_ENABLE_MPI)
   find_package(MPI REQUIRED)
+  option(TTK_ENABLE_MPI_TIME "Enable time measuring for MPI computation" TRUE)
```

Comment:
thanks @eve-le-guillou 
I'd recommend to mark this option as advanced (see `mark_as_advanced` in the same file for examples). thanks!